server/routes/hall.js

```diff
@@ -0,0 +1,19 @@
+import express from "express";
+import { createHall, deleteHall, getHall, getAllHalls} from "../controllers/hall.js";
+import auth from "../middleware/auth.js";
+
+const router = express.Router();
+
+// createHall
+router.post("/createHall", auth, createHall);
+
+// deleteHall
+router.delete("/deleteHall", auth, deleteHall);
```

Comment:
delete hall will take hallName and block as request parameter as discussed in API Endpoints doc